### ISSUE TITLE
How to init additional class and save it for future during the spawn of uwsgi worker?

### ISSUE THREAD

**Issue by idobrodushniy** [unbit/uwsgi#1958]
Hi! 
I want to make some initialization of my DS model for each UWSGI worker during the process of it's spawning.
For example, I want to init class ModelClass during spawn of each worker and then use it in my routes.
Examp:
`keras_model = ModelClass(model_path='models/keras-weights.hdf5')`
Then I would like to get access to this varriable from my routes.
I need to init it in each worker, because of an issue between Tensorflow backend in our Keras model and sharing graphs in memory.
Is it possible to make in UWSGI?

**Comment by xrmx** [comment]
This is more a stack overflow question than an issue. Check out the lazy-apps and postfork decorator.

**Comment by idobrodushniy** [comment]
@xrmx btw, issue can have label "question". 
Thank you very much for your answer, lazy-apps option solved my problem!
